### ISSUE TITLE
Erik Kline Ballot Comments

### ISSUE THREAD

**Issue by SpencerDawkins** [ietf-wg-mops/draft-ietf-mops-streaming-opcons#150]
From @ekline:

> ### S3.4
> 
> * It seems like the final paragraph is attempting to describe a real-world
>   experience.  It would be great if there were a citation for this (but
>   understandable if nothing had been made public).
> 
> ### S3.6
> 
> * RFC 5594 does not appear to contain the word "throttle".  It might not
>   be great to inject this term here?  5594 instead seems to use phrases
>   describing approaches to "congestion management".
> 
> ### S4.4
> 
> * Seems like the introduction of the term "manifest" without any
>   definition, explanation, or reference.  Is there some explanatory text
>   or reference that might be included?

**Comment by SpencerDawkins** [comment]
@acbegen and @GrumpyOldTroll - I'm not sure who wrote this text, except I don't think it was me. 

>And as with other parts of the ecosystem, new technology brings new challenges. For example, with the emergence of ultra-low-latency streaming, responses have to start streaming to the end user while still being transmitted to the cache, and while the cache does not yet know the size of the object. Some of the popular caching systems were designed around cache footprint and had deeply ingrained assumptions about knowing the size of objects that are being stored, so the change in design requirements in long-established systems caused some errors in production. Incidents occurred where a transmission error in the connection from the upstream source to the cache could result in the cache holding a truncated segment and transmitting it to the end user's device. In this case, players rendering the stream often had the video freeze until the player was reset. In some cases the truncated object was even cached that way and served later to other players as well, causing continued stalls at the same spot in the video for all players playing the segment delivered from that cache node.

@ekline was curious if there's a reasonable reference that we could provide for this. 

> **S3.4**
> It seems like the final paragraph is attempting to describe a real-world
> experience. It would be great if there were a citation for this (but
> understandable if nothing had been made public).

**Comment by GrumpyOldTroll** [comment]
Yeah, that was me.  There was not a public reference for this, alas, just a word of mouth anecdote.

**Comment by SpencerDawkins** [comment]
@ekline - I agree with you about your comment on Section 3.6. It happens that I got comments that Section 3.6 and 3.7 didn't need to be nearly as detailed as they were, so they were heavily rewritten. The text where "throttled" was used is now in Section 3.6.1, and the emphasis is now on the cause of unexpected traffic profile changes, rather than the ISP response to the unexpected traffic profile changes in this specific incident, so the text that included "throttled" has been removed.
